app/assets/javascripts/components/form_buttons.js

```diff
@@ -0,0 +1,8 @@
+ManageIQ.angular.app.directive('formButtons', function() {
```

Comment:
could you move this to `directives` instead of `components` please? It's not a component..